### ISSUE TITLE
Get information about on what port is data sended

### ISSUE THREAD

**Issue by Arziel** [getnamo/udp-ue4#7]
I want to create bidirectional UDP trafic and from start i must face with problem created from client hidden after NAT, basically i can't communicate effectively from server back to client, because i don't know on which port is client open connection, is there way to get that opened port information from this plugin? 

Or what is working UC? I can't take this information from server, because port handling at NAT changes the port number.

Example

Server listening at 13000

Client 54786 ---> NAT changes port to 7894 ---> Server:13000

From server I know that client connected via 7894, but i cant tell client that he can listen at 54786 to get message back from me. And sadly i can't even tell them to listen at 7894 because the NAT doesn't perform valid port forwarding back to client itself.

I just don't get it the working solution with plugin. For me it works fine on local and even on some wi-fi routers, but with some workarounds, but for now i must use a mobile hotspot which starts change the port, so i can't use even the server information about connection.

----

TLDR: is there a way to know on what port is open connection for sending, so i can listen at that port?

TY for response.

**Comment by Arziel** [comment]
I think I need access to socket and get on which port is client bounded to sender. From C++ code, it seems like information stored at 

`SenderSocket->GetPortNo()`, does this variable contains port number on which is socket bounded (that generated 50000+), not the one given as server port?

![image](https://user-images.githubusercontent.com/11194301/71769081-b924ae00-2f1c-11ea-9d34-fecfd5827d91.png) 

It is posible to get that number 51878 from BP plugin?

**Comment by getnamo** [comment]
thanks to @Thibault-Brocheton this is now possible https://github.com/getnamo/udp-ue4/pull/18

Pull latest master (0.9.6) to sync and you can find the API here: https://github.com/getnamo/udp-ue4/blob/master/README.md#receiving-on-bound-send-port

**Comment by getnamo** [comment]
Released a build with this feature (1.0.0) https://github.com/getnamo/udp-ue4/releases/tag/1.0.0
